configs/improbable.json

```diff
@@ -32,10 +33,10 @@
     "lvl2": "#docPage h3",
     "lvl3": "#docPage h4",
     "lvl4": "#docPage h5",
-    "text": "#docPage p, #docPage li"
+    "text": "#docPage p, #docPage li, #docPage code"
```

Comment:
Were are documents

Reply:
We don't recommend to do that since it will introduce noise and duplicate within your index:
As for:
```
<p>Check this succeeded by running <code>spatial update</code> in your terminal. You should get the output:</p>
```
It will create one extra records for `<code>spatial update</code>` since the whole one is already a record given that it matches `#docPage p`